src/components/Loader/PersonLink.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const { slugParam } = useParams();
+
+  const { name, sex, born, died, fatherName, motherName, slug } = person;
+
+  const isMother = people.find(p => p.name === motherName);
+  const isFather = people.find(p => p.name === fatherName);
```

Comment:
Would be better to prepare your array of `people` and to add `mother` and `father` to every `person` object